membership/views.py

```diff
@@ -1143,11 +1143,10 @@ def member_reg_form(request, title, pk):
                 # if field invisible
                 if not custom_field['visible']:
                     # remove field
-                    del(custom_fields_displayed[key])
+                    del (custom_fields_displayed[key])
```

Comment:
Does this work with a space between the "del" and the ()?

I suspect that it does, however it does meet proper syntax conventions.